cinq_auditor_domain_hijacking/__init__.py

```diff
@@ -155,7 +156,7 @@ def notify(self, new_issues, existing_issues, fixed_issues):
             try:
                 sender = self.dbconfig.get('from_address', NS_EMAIL)
                 recipients = self.dbconfig.get('email_recipients', self.ns)
-                subject = self.dbconfig.get('hijack_subject', NS_EMAIL, 'Domain hijack possibility detected')
+                subject = self.dbconfig.get('hijack_subject', self.ns)
```

Comment:
Should add the default value here, to ensure that its sent and we don't accidentally send out emails without subjects, cause that'll trigger spam filters